pkg/controller/core/localqueue_controller.go

```diff
@@ -240,8 +229,8 @@ func updateLocalQueueResourceMetrics(queue *kueue.LocalQueue) {
 	recordLocalQueueUsageMetrics(queue)
 }
 
-func (r *LocalQueueReconciler) Generic(e event.GenericEvent) bool {
-	r.log.V(3).Info("Got Workload event", "workload", klog.KObj(e.Object))
+func (r *LocalQueueReconciler) Generic(e event.TypedGenericEvent[*kueue.LocalQueue]) bool {
+	r.log.V(3).Info("Got generic event", "obj", klog.KObj(e.Object), "kind", e.Object.GetObjectKind().GroupVersionKind())
```

Comment:
```suggestion
	r.log.V(2).Info("LocalQueue generic event", "localQueue", klog.KObj(e.Object))
```
IIUC, we already know the kind, right? Just to keep it consistent with other places. Or maybe we should update other places to match this format. WDYT?

Reply:
Good point, let me address it, thanks!

Reply:
Done.